applications/nrf_desktop/doc/ble_discovery.rst

```diff
@@ -26,9 +26,11 @@ Configuration
 Complete the following steps to configure the module:
 
 1. Complete the basic Bluetooth configuration, as described in :ref:`nrf_desktop_bluetooth_guide`.
+   Make sure that both :ref:`CONFIG_DESKTOP_ROLE_HID_DONGLE <config_desktop_app_options>` and :ref:`CONFIG_DESKTOP_BT_CENTRAL <config_desktop_app_options>` options are enabled.
+   The |ble_discovery| is enabled by the :ref:`CONFIG_DESKTOP_BLE_DISCOVERY_ENABLE <config_desktop_app_options>` Kconfig option, which is implied by :ref:`CONFIG_DESKTOP_BT_CENTRAL <config_desktop_app_options>` Kconfig option together with other application modules that are required for HID dongle that forwards data from HID peripherals connected over Bluetooth.
```

Comment:
Maybe technical writers will have an idea how to split that to more manageable pieces.

Reply:
I already split it into a few sentences to improve readability. Still, propositions for further improvements are welcome :)